test/variable.jl

```diff
@@ -219,6 +219,19 @@ end
         @test_throws ErrorException z[end]
     end
 
+    @testset "Key iterators of JuMPContainers" begin
+        m = Model()
+        @variable(m, x[2:3, 1:2])
+        @variable(m, y[2:3])
+        @variable(m, z[-1:1,[:red,"blue"]])
+        @variable(m, w[i=1:2,j=1:2 ; (j+i) % 2 == 1])
+
+        @test all(reshape(collect(keys(x)), :) .== [(2,1), (3,1), (2,2), (3,2)])
```

Comment:
Could we replace `reshape(foo, :)` with `vec(foo)`?

Reply:
didn't remember about vec, sure